### PR TITLE
Add tags

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
Test link: https://cheer-notes-pw51mql11-jay-gadhiya.vercel.app/

functionality : 
- add tags from input by clicking on the tags icon
- add tags from the note by clicking on the tags icon
- adding chips of tags on the note after adding tags
- all notes having tags shown on the tags page divided by tags categories.

### CODE REVIEW

File: src/Pages/Home/Home.jsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ export const HomePage = () => {
     const { noteState, noteDispatch, userNote, setUserNote } = useNote();
     const { authState } = useAuth();
     const [colorPalate, setColorPalate] = useState(false);
+    const [tagBox, setTagBox] = useState(false);
+    let itemTags = [...userNote.tags];
+    let tempTag = "";
```
Comment (jeetbhalani15) [inline src/Pages/Home/Home.jsx:24]: Just a Suggestion you could have used useState for the same.

**Review by jeetbhalani15 — APPROVED** [review]
- Added a suggestion
- I like your UI its clean in Design & in working.
- I also like your tags Modal & code looks so clean and Readable.
Great work @Jay-Gadhiya

**Review by RahullRawat — APPROVED** [review]
Great work. @Jay-Gadhiya 
Your UI looks awesome

**Review by gopalbbharadva — APPROVED** [review]
Functionality working fine, good job.

**Pull request merged**